src/plugins/intel_cpu/tests/functional/shared_tests_instances/snippets/mha.cpp

```diff
@@ -231,55 +164,49 @@ INSTANTIATE_TEST_SUITE_P(smoke_Snippets_MHAEnforceBF16,
                                             ::testing::Values(CPUTestUtils::cpu_bf16_plugin_config)),
                          MHA::getTestCaseName);
 
-INSTANTIATE_TEST_SUITE_P(
-    smoke_Snippets_MHAMulAdd,
-    MHAMulAdd,
-    ::testing::Combine(
-        ::testing::ValuesIn(STATIC_SHAPES({{1, 10, 12, 16}, {1, 10, 12, 16}, {1, 10, 12, 16}})),
-        ::testing::ValuesIn(precision_f32(3)),
-        ::testing::Values(ov::element::f32),
-        ::testing::ValuesIn({false}),  // Need to support True for graph builder in tests
-        ::testing::Values(MHA::default_thread_count),
-        ::testing::Values(1),
-        ::testing::Values(1),
-        ::testing::Values(ov::test::utils::DEVICE_CPU),
-        ::testing::Values(CPUTestUtils::empty_plugin_config)),
-    MHA::getTestCaseName);
+}  // namespace MHADefaultInstances
 
-const auto& inputShapeSelect = STATIC_SHAPES(
-    // without broadcast
-    {{1, 128, 12, 64}, {1, 128, 12, 64}, {1, 12, 128, 128}, {1, 12, 128, 128}, {1, 12, 128, 128}, {1, 128, 12, 64}},
-    {{1, 94, 12, 54}, {1, 94, 12, 54}, {1, 12, 94, 94}, {1, 12, 94, 94}, {1, 12, 94, 94}, {1, 94, 12, 54}},
-    // with broadcast
-    {{1, 128, 12, 64}, {1, 128, 12, 64}, {1, 12, 128, 128}, {1, 12, 1, 1}, {1, 12, 1, 1}, {1, 128, 12, 64}},
-    {{2, 52, 6, 102}, {2, 52, 6, 102}, {1, 6, 52, 52}, {1, 6, 1, 1}, {1, 6, 1, 1}, {2, 52, 6, 102}}
-);
+namespace MHAWOTransposeInstances {
```

Comment:
It's not really convenient to work with these test files with too many test instances: you have use both -DENABLE_CPU_SUBSET_TESTS_PATH and --gtest_filter. 
Do you think it would be better to split this file into several (mha_default, mha_transpose) instead adding the namespaces?

Reply:
Since we decided to not merge INT8/BF16 dynamic MHA support due to perf regression on the large models for now, I created the separate PR which just adds tests: https://github.com/openvinotoolkit/openvino/pull/27169
I splitted the common file "mha.cpp" into small several files. I hope it's more convenient now for debugging and validation now. 
I believe that we can merge that PR asap - it's needed for the validation other features. Thanks!